### PR TITLE
Add raise_exception option to Translator to add workaround for json.decoder.JSONDecodeError

### PR DESCRIPTION
**Pull request by luminousmen**
During translating google API can send you error page back. In my use case this was caused by russian text with emojies. I've this issue with chinese to english translation. Do not really know the underlying reason (I guess it's author's generated token -- cause I get 403 status code), but implement simple workaround for this.

### CODE REVIEW

File: googletrans/client.py [hunk 1]
```diff
@@ -29,20 +30,19 @@ class Translator(object):
...
     :type user_agent: :class:`str`
 
-    :param proxies: proxies configuration. 
-                    Dictionary mapping protocol or protocol and host to the URL of the proxy 
-                    For example ``{'http': 'foo.bar:3128', 'http://host.name': 'foo.bar:4012'}``
+    :param raise_exception: if `True` then raise exception if smth will go wrong
+    :type raise_exception: boolean
     """
 
-    def __init__(self, service_urls=None, user_agent=DEFAULT_USER_AGENT, proxies=None):
```
Comment (ssut) [inline googletrans/client.py]: Could you explain me why you removed proxy support instead of just adding an needed argument? I think you did by mistake.

Comment (luminousmen) [reply googletrans/client.py]: Yeah, it's my mistake - no purpose here

**Comment by luminousmen** [comment]
@kasnitski done, sorry - already forgot about this PR

**Comment by kasnitski** [comment]
@ssut let's merge? what do you think?

**Comment by UserUnknownFactor** [comment]
Why not always rise exception? It's on user to decide what to do if the function breaks in some way so those strange motions with `DUMMY_DATA` and additional parameter seem excessive. 

It would be more clear if custom exception were raised with current service_url, error code and text of error page's `<title>` tag instead.

**Comment by ssut** [comment]
This has been merged into #181.

**Pull request closed without merging**